app/controllers/download_controller.rb

```diff
@@ -39,18 +39,17 @@ def create
   def success; end
 
   def download_json
-    data = open("#{@register.url}/records.json?page-size=5000", &:read)
+    data = download_format("json")
     send_data data, type: "application/json; header=present", disposition: "attachment; filename=#{@register.slug}.json"
   end
 
   def download_csv
-    data = open("#{@register.url}/records.csv?page-size=5000", &:read)
+    data = download_format("csv")
```

Comment:
very minor but we can use single quotes here as we are not doing string interpolation.